src/commands/generate/delta.ts

```diff
@@ -314,8 +316,8 @@ export default class GenerateDelta extends BaseCommand<typeof GenerateDelta> {
         // This is needed because when we re-generate the new profile summary we need the controls
         // to have the new name/Id. So, for each control, modify the control file in the old controls
         // directory with the proper name and Id, than regenerate json profile summary.
-         
-        for (const key in controls) {
+
+        for (const key in controls) { // skipcq: JS-0051
```

Comment:
Where do you reference what each number lint rule means?

Reply:
I tried to find out, but couldn't. I use DeepSource to tell me about the finding